src/platform.ts

```diff
@@ -117,9 +117,10 @@ export class VieramaticPlatform implements DynamicPlatformPlugin {
         return;
       }
     }
+    const tvName = (device.tvName ?? tv.specs.friendlyName)
```

Comment:
it may be simpler here to go something along 

```typescript
if (device.tvName?) { tv.specs.friendlyName = tv.specs.device.tvName) } 
```
as no more more mods needed bellow

Reply:
Agreed, I was being a bit lazy and this way the code is holding the same format as before.